packages/@react-spectrum/s2/src/Button.tsx

```diff
@@ -84,9 +88,6 @@ const button = style<ButtonRenderProps & ButtonStyleProps>({
     ':has([slot=icon]:only-child)': 0
   },
   paddingY: 0,
-  aspectRatio: {
```

Comment:
i thought icon only was always square? are there use cases where they aren't?

Reply:
We supported this in S1 and I haven't seen anything in the designs that explicitly mentions that we've dropped support for it (albeit the designs only show square aspect ratios for icon only). Happy to go either way

Reply:
Didn't Spectrum say we only support square icon buttons in S2?

Reply:
I've been trying to find any mention of this in slack but no luck